webhook_to_fedora_messaging/config/defaults.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Defaults:
+    DEBUG = True
+    TESTING = False
+    PROPAGATE_EXCEPTIONS = None
+    SECRET_KEY = "10505aec23775f6ecb61ded347654282d1be2c673e3b37b0372db6557f39f0ca"
+    PERMANENT_SESSION_LIFETIME = 604800
+    USE_X_SENDFILE = False
+    SERVER_NAME = None
+    APPLICATION_ROOT = "/"
+    SESSION_COOKIE_NAME = "user"
+    SESSION_COOKIE_DOMAIN = None
+    SESSION_COOKIE_PATH = None
+    SESSION_COOKIE_HTTPONLY = True
+    SESSION_COOKIE_SECURE = False
+    SESSION_COOKIE_SAMESITE = None
+    SESSION_REFRESH_EACH_REQUEST = True
+    MAX_CONTENT_LENGTH = None
+    SEND_FILE_MAX_AGE_DEFAULT = None
+    TRAP_BAD_REQUEST_ERRORS = None
+    TRAP_HTTP_EXCEPTIONS = False
+    EXPLAIN_TEMPLATE_LOADING = False
+    PREFERRED_URL_SCHEME = "http"
+    TEMPLATES_AUTO_RELOAD = None
+    MAX_COOKIE_SIZE = None
+
+
+class ProductionDefaults(Defaults):
```

Comment:
This mechanism seems interesting, but I don't see how that's going to work in practice. These are the defaults, right? How is the application going to know if it's run in `Testing` mode or `Production` mode?

Reply:
We can set a sane group of defaults for testing and production environments.

https://flask.palletsprojects.com/en/3.0.x/config/#development-production

Reply:
That's interesting indeed. We'll still have to have a configuration file though, at least for the database URI and URL to Fedora services (stg/prod).